def raise_exception():
    raise TypeError("This is a custom Type Error.")

